tests/ts_util.erl

```diff
@@ -504,7 +504,7 @@ get_optional(N, X) ->
     end.
 
 
--define(DELTA, 1.0e-10).
```

Comment:
Does this mean the results have to be more more accurate? And this would have uncovered the bug in the test?